src/plfa/part2/More.lagda.md

```diff
@@ -1234,7 +1234,7 @@ side to be well typed.
 
 ## Test examples
 
-We repeat the [test examples]({{ site.baseurl }}/DeBruijn/#examples) from Chapter [DeBruijn]({{ site.baseurl }}/DeBruijn),
+We repeat the [test examples]({{ site.baseurl }}/DeBruijn/#examples) from Chapter [DeBruijn]({{ site.baseurl }}/DeBruijn/),
```

Comment:
This is done so the regex in the lua filter's regex matches. It's already done for all other links.